.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
`.eslintignore` file is not required in this case because all these folders are already ignored by default.